resources.md

```diff
@@ -344,11 +343,15 @@ Azure Blob Storage.
 
 </div>
 
-### GCP {#gcp}
+### Google Cloud {#gcp}
```

Comment:
take that change out as well .. and then send another PR to rejig and fast follow .. or do a separate commit in this PR